### PR TITLE
v3.33.1 + Guide on setting up Gradio behind nginx...

### PR DESCRIPTION
**Pull request by abidlabs**
...as we get lets of questions about how to do this.

Closes: #3260 

(Testing) I followed the steps of this Guide to successfully launch: https://www.gradio-staging.com/gradio-demo/

### CODE REVIEW

File: guides/07_other-tutorials/running-gradio-on-your-web-server-with-nginx.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        proxy_set_header Host $host;
+    }
+}
+```
+
+## Run your Gradio app on your web server
+
+1. Before you launch your Gradio app, you'll need to set the `root_path` to be the same as the subpath that you specified in your nginx configuration. This is necessary for Gradio to run on any subpath besides the root of the domain.
+
+    Here's a simple example of a Gradio app with a custom `root_path`:
```
Comment (aliabd) [inline guides/07_other-tutorials/running-gradio-on-your-web-server-with-nginx.md:53]: why is this and the python codeblock under it indented? causing a small formatting issue

**Review by aliabd — APPROVED** [review]
LGTM!

**Comment by abidlabs** [comment]
@aliabd we released 3.33.0 before merging in your other PR refactoring how docs work. I think that caused two issues:

(1) the v3.33.0 html template got published. I've removed it in this PR
(2) the website will no longer build locally since it's expecting a v3.33.0 template to be found in the HF dataset. I think that should get solved when we release v3.30.1 but just wanted to let you know

**Comment by aliabd** [comment]
@abidlabs I've added the v3.33.0 and v3.33.1 templates manually to the dataset. Website builds fine now. 

But I made a mistake and the action to upload to the dataset on a new release isn't working (hence why i also had to manually upload 3.33.1), will fix that now.

**Comment by aliabd** [comment]
(it was a missing indent in the yml file)

**Pull request merged**